ooni/nettests/experimental/tls_handshake.py

```diff
@@ -126,9 +129,11 @@ class HandshakeOptions(usage.Options):
          'Remote host IP address (v4/v6) and port, i.e. "1.2.3.4:443"'],
         ['port', 'p', None,
          'Use this port for all hosts, regardless of port specified in file'],
+        ['sni', None, None, 'Use SNI for Server Name Indication field'],
         ['ciphersuite', 'c', None ,
          'File containing ciphersuite list, one per line'],]
     optFlags = [
+        ['no-sni', None, 'Disable SNI'],
```

Comment:
I don't know what's the best default option from technical standpoint:
  * no sni, contra: that's not what modern browser do
  * sni, contra: fail the test on old python OpenSSL library 
  * based on SNI support, contra: same command-line test launched on different boxes gives different results